tools/syz-cover/syz-cover.go

```diff
@@ -83,8 +96,31 @@ func toolBuildNsHeatmap() {
 	}
 }
 
+func toolFileCover() {
+	dateFrom, dateTo := parseDates()
+	details, err := cover.RendFileCoverage(
+		context.Background(),
+		*flagNamespace,
+		*flagRepo,
+		*flagCommit,
+		*flagForFile,
+		dateFrom,
+		dateTo,
+		cover.RendTextLine,
```

Comment:
Why does this need to be parametrized?

Reply:
Is it enough to parametrize line rendering to get html representation? I would assume the output will need some header/footer, which means the function needs to know what we are rending rather than just accept a callback to render a line.

Reply:
Line will be rendered to <div></div>.
Lines are the strings.Join(lines, "\n").
It will be the valid code representation in html. Web browsers don't need any header/footer to render it.

In our case this rendering result will be used:
1. As a .Body in https://github.com/google/syzkaller/blob/master/dashboard/app/templates/custom_content.html. External code will generate headers/footers.
2. Or will be ajax queried by coverage pages and displayed on the right side of the page.
3. Or will be embedded into one big html file (like we do in cover.html).

I don't know what option will be the first selected but all three scenarios expect the code to be just a <div>, not <html>.